lib/storage/client.js

```diff
@@ -1044,6 +1047,65 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * Make a request object for the create group request
+   * @param {string} name
+   * @param {Array} capabilities
+   * @param {KeyPair} encryptionKeyPair
+   *
+   * @return {Group}
+   */
+  async createGroupRequest(name, capabilities = [], encryptionKeyPair) {
```

Comment:
nit: two things to consider:
- does this method need to be public?
- does this method need to be async?

one additional question:
- does this method belong here?

This last one probably warrants a real-time discussion real quick :)